test/functional/ex_cmds/wviminfo_spec.lua

```diff
@@ -0,0 +1,62 @@
+local helpers, lfs = require('test.functional.helpers'), require('lfs')
+local clear, execute, eq, neq, spawn, nvim_prog, set_session, wait =
+  helpers.clear, helpers.execute, helpers.eq, helpers.neq, helpers.spawn,
+  helpers.nvim_prog, helpers.set_session, helpers.wait
+
+-- Lua does not have a sleep function so we use the system command.  If the
+-- command does not support sub second precision we use math.floor() to get
+-- full seconds.
+local sleep = function(millisec)
+  local sec = millisec / 1000
+  local round = math.floor(sec)
+  if round == 0 then round = 1 end
+  os.execute('sleep '..sec..' || sleep '..round)
```

Comment:
Why use `||` here? `sleep` can't fail, right? Plus, not all shells (or maybe just, annoyingly, fish) support this syntax.